src/Chainweb/Pact/PactService.hs

```diff
@@ -464,45 +479,37 @@ minerReward bh = do
 
 -- | Note: The BlockHeader param here is the PARENT HEADER of the new
 -- block-to-be
+--
+-- Precondition: restoreCheckpointer
```

Comment:
```suggestion
```

`restoreCheckpointer` is not a precondition but provided below as a postcondition of the call to `rewindTo`.